cvrf2csaf/section_handlers/acknowlegments.py

```diff
@@ -0,0 +1,34 @@
+
+from ..common.common import SectionHandler
+
+
+class Acknowledgments(SectionHandler):
+    def __init__(self):
+        super().__init__()
+
+    def _process_mandatory_elements(self, root_element):
+
+        if hasattr(root_element, 'Acknowledgment'):
```

Comment:
Statement has no effect, plain pass in the method is fine. The presence of the Acknowledgement object is verified during ipnut validation.

Anyhow, the separation for mandatory/optional is not very suitable, at first it seemed clever, but for a section like ProductTree (as I implemented it), I think it makes more sense to just separate the "sub-sections" to functions, rather than distinguishing mandatory/optional. 

For now, let's leave it like this, since it has no effect on the functionality, it's just a separation of code.